01/app.js

```diff
@@ -1 +1,9 @@
-console.log('DOM');
\ No newline at end of file
+console.log('DOM');
+
+const el1 = document.querySelector(".comments__item" && ".comments__item--newest")
+
+if(el1) {
+  let arrOfElements = el1.querySelectorAll('span')
+  arrOfElements.forEach(item => item.hasAttribute('data-info'))
+  console.log(arrOfElements.length)
```

Comment:
Tutaj masz ilość wyszukanych `span`-ów - może warto pokombinować z wyszukiwaniem wykorzystując selektor atrybutu - wtedy mogłoby to zadziałać ;)